pyaedt/modeler/cad/component_array.py

```diff
@@ -0,0 +1,718 @@
+            return self.__cells
+
+        if self.__app.settings.aedt_version > "2023.2":  # pragma: no cover
+            self.__array_info_path = self.export_array_info(array_path=None)
+        else:
+            self.__app.save_project()
+
+        self.__cells = [[None for _ in range(self.b_size)] for _ in range(self.a_size)]
+        array_props = self.properties
+        component_names = self.component_names
+        for row_cell in range(0, self.a_size):
+            for col_cell in range(0, self.b_size):
+                self.__cells[row_cell][col_cell] = CellArray(row_cell, col_cell, array_props, component_names, self)
+        return self.__cells
+
+    @property
+    def name(self):
+        """Name of the array."""
+        return self.__name
+
+    @name.setter
+    def name(self, array_name):
+        if array_name not in self.__app.component_array_names:
+            if array_name != self.__name:
+                self.__oarray.SetPropValue("Name", array_name)
+                self.__app.component_array.update({array_name: self})
+                self.__app.component_array_names = list(self.__app.omodelsetup.GetArrayNames())
+                del self.__app.component_array[self.__name]
+                self.__name = array_name
+        else:  # pragma: no cover
+            self.logger.warning("Name %s already assigned in the design", array_name)
+
+    @property
+    def properties(self):
+        """Ordered dictionary of the properties of the component array."""
+        # From 2024R1, array information can be loaded from a CSV
+        if self.__array_info_path and os.path.exists(self.__array_info_path):  # pragma: no cover
+            res = self.parse_array_info_from_csv(self.__array_info_path)
+        else:
+            res = self.__get_properties_from_aedt()
+        return res
+
+    @property
+    def post_processing_cells(self):
+        """Dictionary of each component's postprocessing cells."""
+        if not self.__post_processing_cells:
+            self.__post_processing_cells = {}
+            component_info = {}
+            for row, row_info in enumerate(self.cells, start=1):
+                for col, col_info in enumerate(row_info, start=1):
+                    name = col_info.component
+                    component_info.setdefault(name, []).append([row, col])
+
+            for component_name, component_cells in component_info.items():
+                if component_name not in self.__post_processing_cells.keys() and component_name is not None:
+                    self.__post_processing_cells[component_name] = component_cells[0]
+
+        return self.__post_processing_cells
+
+    @post_processing_cells.setter
+    def post_processing_cells(self, val):
+        if isinstance(val, dict):
+            self.__post_processing_cells = val
+            self.edit_array()
+        else:  # pragma: no cover
+            self.logger.error("Dictionary with component names and cell not correct")
```

Comment:
```suggestion
            self.logger.error("Dictionary with component names and cell is not correct.")
```
Message strings should be complete sentences--which means a verb and concluding punction are required.